maestro/trainer/models/florence_2/entrypoint.py

```diff
@@ -79,6 +84,9 @@ def train(
         metrics=metrics,
         max_new_tokens=max_new_tokens,
         random_seed=random_seed,
+        peft_advanced_params=json.loads(peft_advanced_params)
```

Comment:
let's move `json.loads(peft_advanced_params) if peft_advanced_params is not None else None` above `config` initialization and assign it to a separate variable, before passing here. `json.loads` may also throw exception during parsing. I think we should wrap it in try-except clause and than use `logger.error` to provide users with usefull message.